$NetBSD: patch-client_CMakeLists.txt,v 1.1 2021/05/12 11:20:29 nia Exp $

Ensure that mysql_upgrade gets installed in a client-only build, where
its man page is also installed.

--- client/CMakeLists.txt.orig	2021-02-18 23:22:28.000000000 +0000
+++ client/CMakeLists.txt
@@ -60,7 +60,7 @@ MYSQL_ADD_EXECUTABLE(mysqlimport mysqlim
 SET_SOURCE_FILES_PROPERTIES(mysqlimport.c PROPERTIES COMPILE_FLAGS "-DTHREADS")
 TARGET_LINK_LIBRARIES(mysqlimport ${CLIENT_LIB})
 
-MYSQL_ADD_EXECUTABLE(mysql_upgrade mysql_upgrade.c COMPONENT Server)
+MYSQL_ADD_EXECUTABLE(mysql_upgrade mysql_upgrade.c)
 TARGET_LINK_LIBRARIES(mysql_upgrade ${CLIENT_LIB})
 ADD_DEPENDENCIES(mysql_upgrade GenFixPrivs)
 
